src/videojs-hls.js

```diff
@@ -10,6 +10,9 @@ var
   // a fudge factor to apply to advertised playlist bitrates to account for
   // temporary flucations in client bandwidth
   bandwidthVariance = 1.1,
+
+  // the amount of time to wait between checking the state of the buffer
+  bufferCheckInterval = 500,
```

Comment:
I was thinking 250ms was a bit more frequent than we actually needed, given we call fillBuffer() explicitly whenever we take an action that has a big effect on the buffer.